src/components/ScrollButton.module.css

```diff
@@ -16,11 +16,15 @@
 	text-decoration: none;
 	text-shadow: 0 0 5px var(--color-shadow);
 	z-index: 1;
-	display: none;
+	visibility: hidden;
+	opacity: 0;
+	transition: visibility 0s linear 0.35s, opacity 0.35s linear;
```

Comment:
I tried this out and personally found `0.35s` to be kind of annoyingly slow 😅 sorry for the bad suggestion. What do you think about speeding it up to maybe... 0.2s?

```suggestion
	transition: visibility 0s linear 0.2s, opacity 0.2s linear;
```

Reply:
Yeah sure. I'll make the changes :)

Reply:
Done. Could you review it please.